zou/app/services/shots_service.py

```diff
@@ -1060,13 +1059,17 @@ def get_shot_versions(shot_id):
 
 
 def get_base_entity_type_name(entity_dict):
-    type_name = "Asset"
     if is_shot(entity_dict):
         type_name = "Shot"
+    elif is_scene(entity_dict):
```

Comment:
Can you put the scene tests at then end of the condition list? It occurs less often than the two others. Kitsu users don't have scenes.

Reply:
Makes sense, I fixed it.